src/spyglass/lfp/v1/lfp.py

```diff
@@ -212,7 +165,7 @@ def fetch1_dataframe(self, *attrs, **kwargs):
 
 
 @schema
-class ImportedLFP(dj.Imported):
+class ImportedLFPV1(dj.Imported):
```

Comment:
I'm assuming this is a placeholder for now? If not, it may also be good to add `insert` code within a `make` call with the requisite order since this could be challenging.